doc/user_guide/eds.rst

```diff
@@ -512,11 +511,25 @@ The background can be subtracted from the X-ray intensities with the :py:meth:`~
 Quantification
 --------------
 
-.. versionadded:: 0.8
```

Comment:
I wasn't sure if the tag wanted to be when quantification was added or updated for the two additional quantification methods?